### PR TITLE
Docs rules

### PR DESCRIPTION
**Pull request by xiranyu01**

### CODE REVIEW

File: docs/4-receipts/data-diagnosis/1-intro.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# 关于数据自动采集与诊断
+
+数据自动采集与诊断是针对设备数据提供的运维服务。根据预定义的规则，通过实时监测设备日志，采集目标数据、执行自动诊断，实现设备故障的自动分析与潜在风险的提前探测。
+
+# 基本元素
+
+数据自动采集与诊断流程有三个基本要素，分别为：触发对象、触发条件、触发后的操作。
+- **触发对象**：设备和项目中的日志数据
+- **触发条件**：日志中发生的事件与预定义的规则相匹配
+- **触发后的操作**：上传数据、创建一刻、创建&指派任务等
```
Comment (geoffliu) [inline docs/4-receipts/data-diagnosis/1-intro.md]: bags, too

File: docs/4-receipts/data-diagnosis/2-configuration-rules.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 2
+---
+# 配置数采与诊断规则
+> 每个项目规则单独配置，仅项目管理员可编辑
+
+数采与诊断规则定义触发数据采集与诊断的条件，以及触发后的操作。例如：当设备日志中发生某个事件时，自动采集指定目录下的数据，存储为一条记录，并在该时间点创建一刻。规则的基本结构与标准格式参见后续章节。
+
+在项目中，进入「管理项目-数采与诊断规则」页面，点击【添加规则组】。
```
Comment (geoffliu) [inline docs/4-receipts/data-diagnosis/2-configuration-rules.md]: Link to rule doc

File: docs/4-receipts/data-diagnosis/3-rule-format.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
+---
+sidebar_position: 3
+---
+# 规则的结构与格式
+
+## 规则的使用前提
+
+规则仅对特定格式的数据生效，即：包含消息、时间戳、主题、消息类型的流数据。
+
+其中，支持的时间戳格式如下（[参考文档](https://www.w3schools.com/python/python_datetime.asp)）：
```
Comment (geoffliu) [inline docs/4-receipts/data-diagnosis/3-rule-format.md]: Default, then config

File: docs/4-receipts/data-diagnosis/3-rule-format.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+      - create_moment('Value is not expected')
+enabled: true
+version: 1.0.0
+```
+
+- **name：规则组名称**
+  - 自定义规则组名称，一个规则组中可包含多个规则
+    
+- **when：规则触发条件**
+  - 定义日志中的关键事件，例如日志中包含某个字段
```
Comment (geoffliu) [inline docs/4-receipts/data-diagnosis/3-rule-format.md]: when 中任何一个规则触发以下所有 action

File: docs/4-receipts/data-diagnosis/3-rule-format.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+    
+- **actions：规则触发后的操作**
+  - 定义是否触发上传数据、创建一刻、创建&指派任务等操作
```
Comment (geoffliu) [inline docs/4-receipts/data-diagnosis/3-rule-format.md]: 以上任何条件会触发 **所有** actions

File: docs/4-receipts/data-diagnosis/3-rule-format.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+- **enabled：规则组是否启用**
+  - true 为启用，启用规则组时，设备或项目中的数据触发规则后会执行预定义的操作
+  - false 为禁用，禁用规则组后，设备及项目中的数据不再执行该规则组中的规则
+
+- **version：规则组版本**
+  - 规则组版本随刻行发布而更新
```
Comment (geoffliu) [inline docs/4-receipts/data-diagnosis/3-rule-format.md]: 目前必须是 v1

File: docs/4-receipts/data-diagnosis/3-rule-format.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+<br />
+
+## 规则的标准格式
```
Comment (geoffliu) [inline docs/4-receipts/data-diagnosis/3-rule-format.md]: new page

**Review by geoffliu — CHANGES_REQUESTED** [review]
https://coscene0.feishu.cn/docx/R1bMdQOD8oNf8uxe2OicVITXnwc

**Comment by yujingz** [comment]
@geoffliu im gonna do a merge first for better presentations on the site. if there are more things needs to be tweaked we can do another PR.

**Review by yujingz — APPROVED** [review]

**Pull request merged**